validator/src/main.rs

```diff
@@ -1845,6 +1847,14 @@ pub fn main() {
                 .takes_value(true)
                 .help("The commission validator takes from tips expressed in basis points.")
         )
+        .arg(
+            Arg::with_name("preallocated_bundle_cost")
```

Comment:
i don't see this being used anywhere, maybe not working bc its not piped through to config?

Reply:
that was it, fixed now